cypress/integration/lpa/cases/remove-warning.spec.js

```diff
@@ -0,0 +1,26 @@
+describe('Remove a warning', { tags: ["@lpa", "@smoke-journey"] }, () => {
+  before(() => {
+    cy.loginAs('LPA Manager');
+    cy.createDonor().then(({ id: donorId }) => {
+      cy.createEpa(donorId).then(({ id: lpaId }) => {
+        cy.visit(`/lpa/#/person/${donorId}/${lpaId}`);
+      });
+    });
+  });
+
+  it('should remove the warning', () => {
+    cy.intercept({ method: 'GET', url: '/*/v1/persons/*' }).as('personRequest');
+    cy.intercept({ method: 'GET', url: '/*/v1/persons/*/warnings' }).as('warningsRequest');
+    cy.intercept({ method: 'DELETE', url: '/*/v1/warnings/*' }).as('deleteRequest');
+
+    cy.wait(['@personRequest', '@warningsRequest']);
+
+    cy.contains('.warnings .opg-icon', 'CreateWarning').trigger('mouseover');
```

Comment:
What's this doing?

Reply:
In the UI we have a hand symbol next to the warning, you need to hover over that to reveal the cross icon to delete the warning. Definitely not something we'll be carrying over to the non-angular version

Reply:
![image](https://user-images.githubusercontent.com/146390/170697505-6b4b67f0-0944-467c-a2eb-2ddc583ff779.png)
![image](https://user-images.githubusercontent.com/146390/170697524-4ac0dad9-b6ac-4d39-b965-8b6a6777de81.png)

Reply:
😬